### ISSUE TITLE
Using bidirectional_rnn with lstm, returning sequences and with dropout

### ISSUE THREAD

**Issue by jrzaurin** [tflearn/tflearn#885]
I am trying to code a bidirectional lstm. In keras this looks like this: 

    x = Bidirectional(LSTM(128, dropout=0.2, recurrent_dropout=0.2,return_sequences=True))(x)

following the example [here](https://github.com/tflearn/tflearn/blob/master/examples/nlp/bidirectional_lstm.py)  and the recommendations at #210 I started with this code

     net = bidirectional_rnn(net, BasicLSTMCell(128), BasicLSTMCell(128))

with the idea of  changing `BasicLSTMCell`, to `lstm`, given the fact that in the documentation, one can read: 

`"Build a bidirectional recurrent neural network, it requires 2 RNN Cells to process sequence in forward and backward order. Any RNN Cell can be used i.e. SimpleRNN, LSTM, GRU... with its own parameters. But the two cells number of units must match."` 

First, `BasicLSTMCell` does not have `dropout` parameter or `return_seq ` but `tflearn.layers.recurrent.lstm` does. However, `lstm` requires that you pass the incoming tensor, in this case `net`, so I thought about using this: 

    net = bidirectional_rnn(net, lstm(net, 128), lstm(net, 128))

but this returns an error because `lstm` has no `._n_units` attribute, and `bidirectional_rnn` needs that attribute. So, in summary: the cell I want to pass to the bidirectional layer is the following: 

    lstm(net, n_units=128, dropout=(0.2,0.2), return_seq=True)

Anyone can help?  

Thanks!
